src/components/App.jsx

```diff
@@ -41,14 +41,23 @@ const ContactsApp = props => {
 
   const setStateOfServiceToLaunch = useCallback(
     async () => {
-      const serviceToLaunch = await fetchNormalizedServiceByName(
-        client,
-        'keepIndexFullNameAndDisplayNameUpToDate'
-      )
-      setServiceToLaunch(serviceToLaunch)
-      setHasServiceBeenLaunched(
-        get(serviceToLaunch, 'current_state.last_success', '').length > 0
-      )
+      try {
+        const serviceToLaunch = await fetchNormalizedServiceByName(
+          client,
+          'keepIndexFullNameAndDisplayNameUpToDate'
+        )
+
+        setServiceToLaunch(serviceToLaunch)
+        setHasServiceBeenLaunched(
+          get(serviceToLaunch, 'current_state.last_success', '').length > 0
+        )
+      } catch (e) {
```

Comment:
If fetchNormalizedServiceByName returns null in case of a problem, is the catch here still relevant?

Reply:
If the wrapped code will not throw, the catch is not relevant anymore.